File: Entities/Models/AnswerVotersList.cs
namespace QaNet.Entities.Models
{
  public class AnswerVotersList : BaseEntity
  {
    public string VoterId { get; set; }

    public int AnswerId { get; set; }

    public int Vote { get; set; }

    public User User { get; set; }

    public Answer Answer { get; set; }
  }
}

File: models/answer_voters_list.go
package models

// AnswerVotersList is the model for the answer_voters_list table.
type AnswerVotersList struct {
	AnswerID int64  `json:"AnswerId"`
	VoterID  string `json:"voterId"`
	Vote     int    `json:"vote"`
	Base
}
